tomviz/python/WienerFilter.py

```diff
@@ -0,0 +1,48 @@
+def transform_scalars(dataset, SX=0.5, SY=0.5, SZ=0.5, noise=15.0):
+    """Deblur Images with a Weiner Filter."""
+
+    from tomviz import utils
+    import numpy as np
+    from numpy import exp, square, pi
+    from scipy.fftpack import fftn, ifftn, fftshift
```

Comment:
These should probably be moved to the top.

Reply:
I mean that the imports are usually place at the top of the file rather than inside the function.

Reply:
Ok, I see you were just copying the pattern from the other  operators, so lets leave it as it is.